### ISSUE TITLE
Better naming idea for the carbon

### ISSUE THREAD

**Issue by emreakay** [carbon-language/carbon-lang#1694]
hi,

due to carbon is successor of C++,
its name should be C+++.

what do you think about that idea?

**Comment by guiofsaints** [comment]
That makes sense!

**Comment by MarcoCicognani** [comment]
I think that the name Carbon doesn't sound bad, but use `.carbon` as extension is really goofy

**Comment by emreakay** [comment]
@MarcoCicognani extension can be .cppp

**Comment by xianjinfu** [comment]
C+++ is not friendly enough  for non-english native speakers, coz it's a little confusing to tell c plus plus from c plus plus plus...

**Comment by MarcoCicognani** [comment]
@emreakay personally I use `.cc` and `.hh`

**Comment by dgrelaud** [comment]
+1 

Or just `C+`,  because the perfection is achieved when there's nothing left to take away. Antoine de Saint-Exupery.

`Carbon language` is close to my Carbone templating language 😊
- https://github.com/carboneio/carbone
- https://carbone.io/documentation.html#philosophy  

When David meets Goliath, uh Google  😅.

**Comment by guiofsaints** [comment]
> 

@emreakay  why not `.c3p`?  LOL

Carbon is a good name, the extension could be:` .co2 ` or `.c6`

**Comment by guiofsaints** [comment]
How about `C6 Lang`?

**Comment by omerfarukz** [comment]
How about C3+?

**Comment by smassarolucas** [comment]
How about ++C++? This way we increment before evaluation giving a sentiment of progress, put an emphasis on the 'C' aspect of things and the extension can be `.ppcpp`

**Comment by jonmeow** [comment]
I'm closing this issue: don't use Carbon's forums for jokes, it's off-topic. Please review our [Code of Conduct](https://github.com/carbon-language/carbon-lang/blob/trunk/CODE_OF_CONDUCT.md) and stay on-topic.

**Comment by chandlerc** [comment]
BTW, just to follow-up on one very specific concern from @dgrelaud:

> `Carbon language` is close to my Carbone templating language 😊
> 
> * [carboneio/carbone](https://github.com/carboneio/carbone)
> * [carbone.io/documentation.html#philosophy](https://carbone.io/documentation.html#philosophy)

Very sorry that we ended up so close to this -- this didn't show up in any of our research when we picked the name I'm afraid.

Hopefully the areas are far enough to not cause too much lasting confusion. Also feel free to reach out to me via email/discord/etc if I can help in any way to help. Don't want to re-open this bug, but happy to dig into this specific concern.
